ludwig/models/llm.py

```diff
@@ -66,6 +91,35 @@ def __init__(
 
         clear_data_cache()
 
+    def to_device(self, device):
+        device = torch.device(device)
+        if device == self.curr_device:
+            logger.warning(f"LLM already on device '{device}'. Skipping device placement step.")
+            return self
+        else:
+            logger.info(f"Moving LLM from '{self.curr_device}' to '{device}'.")
+
+        model_kwargs = {}
+        if device == torch.device("cuda"):
+            num_gpus = torch.cuda.device_count()
+            model_kwargs.update(
+                dict(
+                    low_cpu_mem_usage=True,
```

Comment:
TODO here about making this configurable in the future. I would also reference FastChat as the source of these params.

Reply:
Done.